src/ubx.h

```diff
@@ -202,6 +204,15 @@
 #define UBX_TX_CFG_NAV5_MASK		0x0005		/**< Only update dynamic model and fix mode */
 #define UBX_TX_CFG_NAV5_FIXMODE		2		/**< 1 2D only, 2 3D only, 3 Auto 2D/3D */
 
+/* TX CFG-RST message contents
+ * Note: not used with protocol version 27+ anymore
```

Comment:
Can you add a TODO in the code or use the new message if `_proto_ver_27_or_higher` is set?

Reply:
Removed the comment - it was a copy/paste error and it was not vaild for CFG-RST